src/components/ToolTip/ToolTip.less

```diff
@@ -0,0 +1,323 @@
+	}
+	&-danger&-left:after {
+		border-left-color: @featured-color-danger-backgroundColor;
+	}
+	&-danger&-left:before {
+		border-left-color: @featured-color-danger-borderColorLite;
+	}
+	&-danger&-right:after {
+		border-right-color: @featured-color-danger-backgroundColor;
+	}
+	&-danger&-right:before {
+		border-right-color: @featured-color-danger-borderColorLite;
+	}
+	&-danger&-down:after {
+		border-bottom-color: @featured-color-danger-backgroundColor;
+	}
+	&-danger&-down:before {
+		border-bottom-color: @featured-color-danger-borderColorLite;
+	}
+	&-danger {
+		border-color: @featured-color-danger-borderColorLite;
+		background-color: @featured-color-danger-backgroundColor;
+	}
+
+	&-title {
+		font-size: @size-L;
+		font-weight: bold;
+		margin: 0 0 @size-XS;
+		line-height: 1em;
+	}
+
+	&:after, &:before {
+		content: ' ';
+		height: 0;
+		width: 0;
+		position: absolute;
+		pointer-events: none;
+		border: 7px solid transparent;
+	}
+
+	&:after {
+		border-width: @tooltip-pointer-inner-width;
+	}
+
+	&:before {
+		border-width: @tooltip-pointer-outer-width;
+	}
+
+	&-up:after,
+	&-up:before {
+		top: 100%;
+	}
+
+	&-up:after {
+		margin-left: -@tooltip-pointer-inner-width;
+	}
+
+	&-down:after,
+	&-down:before {
+		bottom: 100%;
+	}
+
+	&-up&-center:after,
+	&-up&-center:before,
+	&-down&-center:after,
+	&-down&-center:before {
+		left: 50%;
+	}
+
+	&-up&-center:before,
+	&-down&-center:before {
+		margin-left: -@tooltip-pointer-outer-width;
+	}
+
+	&-up&-center:after,
+	&-down&-center:after {
+		margin-left: -@tooltip-pointer-inner-width;
+	}
+
+	&-up&-start:after,
+	&-down&-start:after {
+		margin-left: 1px;
+	}
+
+	&-up&-end:after,
+	&-down&-end:after {
+		margin-right: 1px;
+	}
+
+	&-down&-start:after,
+	&-down&-start:before,
+	&-up&-start:after,
+	&-up&-start:before {
+		left: @tooltip-pointer-offset;
+	}
+
+	&-down&-end:after,
+	&-down&-end:before,
+	&-up&-end:after,
+	&-up&-end:before {
+		right: @tooltip-pointer-offset;
+	}
+
+	&-left:after,
+	&-left:before {
+		left: 100%;
+	}
+
+	&-right:after,
+	&-right:before {
+		right: 100%;
+	}
+
+	&-left&-center:after,
+	&-left&-center:before,
+	&-right&-center:after,
+	&-right&-center:before {
+		top: 50%;
+	}
+
+	&-left&-center:before,
+	&-right&-center:before {
+		margin-top: -@tooltip-pointer-outer-width;
+	}
+
+	&-left&-center:after,
+	&-right&-center:after {
+		margin-top: -@tooltip-pointer-inner-width;
+	}
+
+	&-left&-start:after,
+	&-left&-start:before,
+	&-right&-start:after,
+	&-right&-start:before {
+		top: @tooltip-pointer-offset;
+	}
+
+	&-left&-end:after,
+	&-left&-end:before,
+	&-right&-end:after,
+	&-right&-end:before {
+		bottom: @tooltip-pointer-offset;
+	}
+
+	&-left&-start:after,
+	&-right&-start:after {
+		margin-top: 1px;
+	}
+
+	&-left&-end:after,
+	&-right&-end:after {
+		margin-bottom: 1px;
+	}
+
+	&-close {
+		position: absolute;
+		top: 6px;
+		right: 6px;
+		cursor: pointer;
+		fill: fade(@color-black, 50%);
```

Comment:
Use `@color-gray-50` for this so we can keep track of all the different grays we're using